consoles/sshVirtsh.pm

```diff
@@ -352,18 +352,20 @@ sub provide_image_vmware_in_ds ($self, $input_file, $vmware_openqa_datastore, %a
     # check image is present
     my $cmd = <<~"EOF";
     $debug
+    IN=$input_file
     if test -e "$dest_image"; then
         echo "Waiting while $input_file is loading:"
         while ps -v | grep -E "cp .*$baseimage|xz .*$basefile"|grep -v grep
             do sleep 5; done
         echo "VMware image $dest_image ready"
-    elif [[ "$input_file" == *.xz ]]; then 
+    elif [[ \${IN: -3} == '.xz' ]]; then 
```

Comment:
It would be better to stick to POSIX shell syntax. I believe `[[ ]]` is not POSIX, but I might be wrong

Reply:
this check now works fine both on bash as sh:
should I use `test ...` ?

Reply:
Done

Reply:
CI pass ok